Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -259,13 +264,15 @@ def calculate_indicator_score(self, risk_from_feed):
         Returns:
             int. The indicator's Dbot score
         """
-        dbot_score = 0
         risk_from_feed = int(risk_from_feed)
-        if risk_from_feed >= self.threshold or risk_from_feed >= 65:
+        if risk_from_feed >= self.malicious_threshold:
             dbot_score = 3
-        elif risk_from_feed >= 5:
+        elif risk_from_feed >= self.suspicious_threshold:
             dbot_score = 2
-
+        elif risk_from_feed == 0:
+            dbot_score = 1
+        else:
+            dbot_score = 0
```

Comment:
What risk_from_feed can be here? a negative value?

Reply:
The final else statement covers the risk range 0<risk_from_feed<suspicious_threshold. I could change the conditionals to this for greater clarity: 
```
        if risk_from_feed >= self.malicious_threshold:
            dbot_score = 3
        elif risk_from_feed >= self.suspicious_threshold:
            dbot_score = 2
        elif risk_from_feed > 0:
            dbot_score = 0
        else:
            #risk_from_feed == 0
            dbot_score = 1 
```

Reply:
Let's do that.